packages/flutter/lib/src/rendering/object.dart

```diff
@@ -185,10 +185,20 @@ class PaintingContext {
 
   static final Paint _disableAntialias = new Paint()..isAntiAlias = false;
 
-  /// Push a performance overlay.
```

Comment:
when referring to arguments that don't map to properties, use "`" rather than "[]".